client/style.scss

```diff
@@ -169,3 +169,24 @@
 	.components-tooltip {
 	display: none;
 }
+
+img {
+	&.payment-method__icon {
+		outline: 1px solid rgba( 0, 0, 0, 0.25 );
```

Comment:
@bradyv Could answer better to this question as the credit for CSS goes to him.

AFAIU, the issue with the border was that it's is drawn around the asset which created the inconsistencies between CC icon (which had the individual borders baked into the asset) and other assets. 

With the new grid approach that might not be the problem anymore. 

Luckily, we now have single source of truth for the border CSS and it can be easily adjusted.

Reply:
@mgascam Basically exactly what @KarlisJ  said. `outline` allows us to set a negative offset that draws the stroke on top of the icon. Using `border` required additional mockup that made it much more difficult to scale to different icon sizes.

Beyond just inconsistencies between assets this also gives us a much nicer display as the stroke is clearly visible for all icons regardless of icon colour or the background colour they're displayed on.

Reply:
Thanks for clarification @bradyv @KarlisJ